content/kubermatic/v2.26/architecture/compatibility/os-support-matrix/_index.en.md

```diff
@@ -15,26 +15,24 @@ The following operating systems are currently supported by Kubermatic:
 * RHEL beginning with 8.0 (support is cloud provider-specific)
 * Flatcar (Stable channel)
 * Rocky Linux beginning with 8.0
-* CentOS beginning with 7.4 excluding stream versions
 * Amazon Linux 2
-
+**Note:** CentOS was removed as a supported OS in KKP 2.26.3
```

Comment:
Can we make this a warning please, to highlight it a bit more? You can see examples of that here: https://github.com/kubermatic/docs/blob/main/content/kubermatic/v2.26/installation/install-kkp-ce/_index.en.md?plain=1#L29.